crates/cdk-phoenixd/README.md

```diff
@@ -0,0 +1,46 @@
+# cdk-phoenixd
+
+## Run phoenixd
+
+The `phoenixd` node is included in the cdk and needs to be run separately.
+Get started here: [Phoenixd Server Documentation](https://phoenix.acinq.co/server/get-started)
+
+## Start Phoenixd
+
+By default, `phoenixd` will run with auto-liquidity enabled. While this simplifies channel management, it makes fees non-deterministic, which is not recommended for most scenarios. However, it is necessary to start with auto-liquidity enabled in order to open a channel and get started.
+
+Start the node with auto-liquidity enabled as documented by [Phoenixd](https://phoenix.acinq.co/server/get-started):
+```sh
+./phoenixd
+```
+
+> **Note:** By default the `auto-liquidity` will open a channel of 2m sats depending on the size of mint you plan to run you may want to increase this by setting the `--auto-liquidity` flag to `5m` or `10m`.
+
+## Open Channel
+
+Once the node is running, create an invoice using the phoenixd-cli to fund your node. A portion of this deposit will go to ACINQ as a fee for the provided liquidity, and a portion will cover the mining fee. These two fees cannot be refunded or withdrawn from the node. More on fees can be found [here](https://phoenix.acinq.co/server/auto-liquidity#fees). The remainder will stay as the node balance and can be withdrawn later.
+```sh
+./phoenix-cli createinvoice \
+    --description "Fund Node" \
+    --amountSat 30000
```

Comment:
This needs to be much higher or constant new channels (splice ins) will be needed. Maybe should be the max size of the mint you want to operate?